src/SDKs/Search/DataPlane/Microsoft.Azure.Search/generate.cmd

```diff
@@ -0,0 +1,27 @@
+::
```

Comment:
We are using a specific version of AutoRest for the DataPlane because the first non-NuGet versions had some issues that we couldn't work around. We haven't had the time to kick the tires on newer versions since then. It's important that we control the pace of these updates, because we have a lot of hand-written code right now that has to track changes in the AutoRest-generated code.

Please revert this until my team has the bandwidth to take these changes.